pages/configuration/configuration-settings.mdx

```diff
@@ -9,32 +9,182 @@ import { Steps } from 'nextra/components'
+
+        ```bash
+        sudo systemctl start memgraph
+        ```
+    </Tab>
+</Tabs>
+
+
+
+### Change configuration settings during runtime
+
+Memgraph contains settings that can be modified during runtime using a Cypher query.
+Some runtime settings are persisted between multiple runs, while others will
+fallback to the value of the command-line argument.
+
+| Setting name               | Description                                                                                   | Persistent between runs |
+|----------------------------|-----------------------------------------------------------------------------------------------|-------------------------|
+| organization.name          | Name of the organization using the instance of Memgraph (used for verifying the license key). | yes                     |
+| enterprise.license         | License key for Memgraph Enterprise.                                                          | yes                     |
+| server.name                | Bolt server name.                                                                             | yes                     |
+| query.timeout              | Maximum allowed query execution time. Value of 0 means no limit.                              | yes                     |
+| log.level                  | Minimum log level. Allowed values: TRACE, DEBUG, INFO, WARNING, ERROR, CRITICAL.              | no                      |
+| log.to_stderr              | Log messages go to `stderr` in addition to `logfiles`.                                        | no                      |
+| cartesian-product-enabled  | Enforces cartesian product operator during query matching.                                    | no                      |
+
+All settings can be fetched by calling the following query:
 
-You can check the current configuration by using the following query (privilege
-level `CONFIG` is required):
 ```opencypher
-SHOW CONFIG;
+SHOW DATABASE SETTINGS;
+```
+
+To check the value of a single setting, you can use a slightly different query:
+
+```opencypher
+SHOW DATABASE SETTING "setting.name";
 ```
 
+If you want to change a value for a specific setting, following query should be used:
+
+```opencypher
+SET DATABASE SETTING "setting.name" TO "some-value";
+```
+
+
+### Changing the configuration settings via file 
+
+Additional configurations can be specified by including another
+configuration file, in a file pointed to by the `MEMGRAPH_CONFIG` environment
+variable.
+
+You can define additional configuration files in the main configuration file or
+within a Docker command in the terminal. Additional files are processed after
+the main configuration file, and they override the main configuration file.
+Additional configuration files are specified with the `--flag-file` flag.
+
+Example:
+
+ `--flag-file=another.conf`
+
+
+
+### Multitenancy and configuration 
+
```

Comment:
What do you think about the title before audit log -> List of configuration settings
And then Audit log until Other all under that title/section?

Reply:
Also, I feel like env variables section is then there out of nowhere with no additional context. Can you update that as well? Maybe add a section similar to init flags on how to use those env vars there?

Reply:
I have updated the title structure. 

> Also, I feel like env variables section is then there out of nowhere with no additional context. Can you update that as well? Maybe add a section similar to init flags on how to use those env vars there?

I have added a guide but this means adding a docker and docker-compose example.